src/routes/authRoutes.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from "express";
+import { login, register } from "../controllers/authController";
+
+const router = Router();
+router.get("/login", login);
```

Comment:
I think login should also be POST request as it hold critical data like password.

Reply:
Yes. Thank you for pointing that out. That totally slipped through my eyes.